src/routers/handlers/individual-psc-list/individualPscListHandler.ts

```diff
@@ -57,6 +59,11 @@ export class IndividualPscListHandler extends GenericHandler<IndividualPscListVi
         }
 
         const allPscDetails = this.getViewPscDetails(individualPscList, lang);
+        const allSuperSecure = allPscDetails.every((psc) => psc.pscKind === PSC_KIND_TYPE.SUPER_SECURE);
+        const allCeased = allPscDetails.every((psc) => psc.pscCeasedOn != null);
+
+        const exclusivelySuperSecure = allPscDetails.length > 0 && (allSuperSecure && !allCeased);
+        const showNoPscsMessage = allPscDetails.length === 0 || allCeased;
 
```

Comment:
I haven't got my head round this, but when I have all Super Secure PSC's (not ceased), then I don't get sent to the Super Secure Stop screen as expected.